web/src/pages/Settings/Settings.jsx

```diff
@@ -90,7 +76,13 @@ const Settings = ({ openPage }) => {
     const [isSavedUrl, setIsSavedUrl] = React.useState(IsSaved());
 
     if (openPage != "") {
-        return <Page openPage={openPage} />;
+        return (
+            <div className="settings-page-container">
+                <StopProp>
+                    <CorePage openPage={openPage} />
+                </StopProp>
+            </div>
+        );
```

Comment:
`Modal` accomplishes a couple of things:
* `event.stopPropogate()` during `onClose()`
* wraps all children in a `StopProp`
* applies a decent chunk of CSS from `Modal.scss`
* tags with a `modal_container` CSS class, which is used in the above class but could also be added to  elsewhere

can you validate that in all the `CorePage` cases none of this is used or necessary? that's my hesitancy to rip this out - it may fix the spacing issue, but I'm unconvinced we can test the surface area changes ripping it out entails in short order, before trying to press a release.